lib/src/main/java/com/orange/ods/compose/theme/OdsTheme.kt

```diff
@@ -97,7 +96,7 @@ fun OdsTheme(
         MaterialTheme(
             colors = colors.materialColors
         ) {
-            Surface(color = colors.background, content = content)
+            content()
```

Comment:
FYI, although the fix is OK for me, issue #621 is now back.

But I am not sure it is truly an issue, as this works correctly when adding a `Surface` around the `OdsTextField`. What do you think?

Reply:
I agree with you, I'm not sure this is an issue.